darshan-util/pydarshan/tests/conftest.py

```diff
@@ -8,3 +20,15 @@
 
 def pytest_configure():
     pytest.has_log_repo = has_log_repo
+
+@pytest.fixture
+def log_repo_files():
+    # provide a convenient way to access the list
+    # of all *.darshan log files in the logs repo,
+    # returning an empty list in the absence of the
+    # logs repo package and a list of absolute file
+    # paths to the logs otherwise
+    if not has_log_repo:
+        return []
```

Comment:
honestly, it might be better if this case is not handled at all, and we simply get an error if someone uses the log_repo_files fixture without the appropriate `pytest` skip guards

Reply:
I've made this adjustment/revision